tests/FilterRule/SlugTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+namespace Particle\Tests\Filter\FilterRule;
+
+use Particle\Filter\Filter;
+
+class SlugTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Filter
+     */
+    protected $filter;
+
+    /**
+     * Prepare the filter
+     */
+    public function setUp()
+    {
+        $this->filter = new Filter();
+    }
+
+    /**
+     * @dataProvider getSlugResults
+     * @param string $value
+     * @param string $filteredValue
+     * @param string $field
+     * @param string $fieldValue
+     */
+    public function testSlugFilterRule($value, $filteredValue, $field, $fieldValue)
+    {
+        $this->filter->value('test')->slug($field);
+
+        $result = $this->filter->filter([
+            'test' => $value,
+            $field => $fieldValue,
+        ]);
+
+        $this->assertEquals($filteredValue, $result['test']);
```

Comment:
Oh, ok @localheinz ! Well i guess we can, yes. I honestly copied and passed another test case and reused `assertEquals`. The slug filter should always return a `string` so it'd work too.